install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/values.yaml

```diff
@@ -6,3 +6,8 @@ replicaCount: 1
 image: sidecar_injector
 enableNamespacesByDefault: false
 nodeSelector: {}
+
+# If true, webhook or istioctl injector will rewrite PodSpec for liveness
+# health check to redirect request to sidecar. This makes liveness check work
+# even when mTLS is enabled.
+rewriteAppHTTPProbe: false
```

Comment:
why not turn it on always if you are doing other things?

Reply:
I left a few thibgs here for discovery, very few. But ok removing it from here.
No need to define defaults 2 times.